sdk/nodejs/location.ts

```diff
@@ -12,38 +12,40 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-export let EastAsia:           Location = "eastasia";
-export let SouthEastAsia:      Location = "southeastasia";
-export let CentralUS:          Location = "centralus";
-export let EastUS:             Location = "eastus";
-export let EastUS2:            Location = "eastus2";
-export let WestUS:             Location = "westus";
-export let WestUS2:            Location = "westus2";
-export let NorthCentralUS:     Location = "northcentralus";
-export let SouthCentralUS:     Location = "southcentralus";
-export let NorthEurope:        Location = "northeurope";
-export let WestEurope:         Location = "westeurope";
-export let JapanWest:          Location = "japanwest";
-export let JapanEast:          Location = "japaneast";
-export let BrazilSouth:        Location = "brazilsouth";
-export let AustraliaEast:      Location = "australiaeast";
-export let AustraliaSouthEast: Location = "australiasoutheast";
-export let AustraliaCentral:   Location  = "australiacentral";
-export let AustraliaCentral2:  Location = "australiacentral2";
-export let SouthIndia:         Location = "southindia";
-export let CentralIndia:       Location = "centralindia";
-export let WestIndia:          Location = "westindia";
-export let CanadaCentral:      Location = "canadacentral";
-export let CanadaEast:         Location = "canadaeast";
-export let UKSouth:            Location = "uksouth";
-export let UKWest:             Location = "ukwest";
-export let WestCentralUS:      Location = "westcentralus";
-export let KoreaCentral:       Location = "koreacentral";
-export let KoreaSouth:         Location = "koreasouth";
-export let FranceCentral:      Location = "francecentral";
-export let FranceSouth:        Location = "francesouth";
-export let SouthAfricaNorth:   Location = "southafricanorth";
-export let SouthAfricaWest:    Location = "southafricawest";
+export module Locations {
```

Comment:
Why capital `L`?

Reply:
To go with the convention we have used elsewhere in other providers - @jen20 thoughts here?

Reply:
@mikhailshilkov this comes from pulumi-aws#599 btw - I just spoke to @jen20 about it

Reply:
The pattern comes from pulumi/pulumi-aws#599 - probably worth sticking with at this point?